package.json

```diff
@@ -69,11 +69,13 @@
   ],
   "sideEffects": false,
   "type": "module",
-  "exports": "./index.js",
+  "exports": {
+    "types": "./types/exports.d.ts",
+    "default": "./lib/index.js"
+  },
```

Comment:
Is it correct that the `types` field here is only needed because you want to overwrite `lib/index.d.ts`, for consumers of `'react-markdown'`?